packages/material-ui/src/List/List.spec.tsx

```diff
@@ -1,15 +1,14 @@
 import * as React from 'react';
 import List from '@material-ui/core/List';
 
-// use other components
+// custom host
```

Comment:
Probably a bit redundant considering we have the docs but since we also test the callback and it had an issue I'd say we keep it.